File: BBIS.Application/DTOs/Lookup/LookupDto.cs
namespace BBIS.Application.DTOs.Lookup
{
    public class LookupDto
    {
        public int LookupId { get; set; }
        public string LookupKey { get; set; }
        public string Description { get; set; }

        public List<LookupOptionDto> LookupOptions { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/Lookups/LookupDto.ts
import { ILookupOptions } from "./LookupOptions";

export interface ILookup {
  LookupId: number,
  LookupKey: string,
  Description: string,
  LookupOptions: Array<ILookupOptions>
}

export class Lookup implements ILookup{
  LookupId: number = 0;
  LookupKey: string = "";
  Description: string = "";
  LookupOptions: Array<ILookupOptions> = [];
}